cli/commands/scripts_update_config_v2.go

```diff
@@ -343,6 +343,14 @@ func newScriptsUpdateConfigV2Cmd(ec *cli.ExecutionContext) *cobra.Command {
 			}
 			ec.Spinner.Stop()
 			ec.Logger.Infoln("Updated config to version 2")
+
+			if _, err = os.Stat(filepath.Join(ec.MigrationDir, "metadata.yaml")); err == nil || os.IsExist(err) {
```

Comment:
There will be a file pointer only when we can successfully "stat" a file right? which means we are not interested in the error here, so maybe something like this also would work?
```
if f, _ = os.Stat(filepath.Join(ec.MigrationDir, "metadata.yaml")); f {
    // do stuff
}
```

Reply:
yeah